### ISSUE TITLE
ReactPrerfTools: 15.1.0-alpha.1 

### ISSUE THREAD

**Issue by borisyankov** [facebook/react#6742]
I was testing latest refactor of PerfTools.
When using printOperations() often (but not always) I get 

```
Uncaught TypeError: Cannot read property 'displayName' of undefined
```

in getOperations() on the line:

```
var {displayName, ownerID} = treeSnapshot[instanceID];
```

**Comment by gaearon** [comment]
Is there any way you can find a test case reproducing this? That would be really helpful!

**Comment by borisyankov** [comment]
This happens in a pretty big app, not always but relatively often... hard to isolate something simple.

In could pinpoint it to: in the iteration of operations.forEach, the operation variable has value of:

     {instanceID: 0, type: "replace with", payload: "P"}

...and operation[instanceID] is undefined.

If you can not reproduce or figure it out from that, the app while big, is open sourced and available at:
https://github.com/binary-com/binary-next-gen

**Comment by gaearon** [comment]
This probably happens when a composite switches from rendering `null` to rendering a DOM tag.

**Comment by gaearon** [comment]
Fixed by #6752. Big thanks for reporting.
